2021-04-26 21:04:14,268 INFO    MainThread:2878 [wandb_setup.py:_flush():69] setting env: {}
2021-04-26 21:04:14,268 INFO    MainThread:2878 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-26 21:04:14,268 INFO    MainThread:2878 [wandb_init.py:_log_setup():336] Logging user logs to /home/gavin/Documents/GitHub/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210426_210414-qk3ra2fd/logs/debug.log
2021-04-26 21:04:14,269 INFO    MainThread:2878 [wandb_init.py:_log_setup():337] Logging internal logs to /home/gavin/Documents/GitHub/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210426_210414-qk3ra2fd/logs/debug-internal.log
2021-04-26 21:04:14,270 INFO    MainThread:2878 [wandb_init.py:init():369] calling init triggers
2021-04-26 21:04:14,270 INFO    MainThread:2878 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {'model': 'lf', 'dataset': 'type10', 'type_num': 10, 'n_features': 27, 'n_hidden': 128, 'seq_len': 672, 'n_layers': 2, 'out_features': 96, 'do': 0.2, 'batch_size': 24, 'client_optimizer': 'adam', 'lr': 0.0005, 'wd': 0.001, 'epochs': 5, 'client_num_in_total': 3, 'client_num_per_round': 3, 'comm_round': 10, 'frequency_of_the_test': 1, 'gpu': 0, 'ci': 0}
2021-04-26 21:04:14,270 INFO    MainThread:2878 [wandb_init.py:init():418] starting backend
2021-04-26 21:04:14,271 INFO    MainThread:2878 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-26 21:04:14,296 INFO    MainThread:2878 [backend.py:ensure_launched():123] starting backend process...
2021-04-26 21:04:14,302 INFO    MainThread:2878 [backend.py:ensure_launched():128] started backend process with pid: 2890
2021-04-26 21:04:14,303 INFO    MainThread:2878 [wandb_init.py:init():423] backend started and connected
2021-04-26 21:04:14,333 INFO    MainThread:2878 [wandb_init.py:init():465] updated telemetry
2021-04-26 21:04:14,333 INFO    MainThread:2878 [wandb_init.py:init():484] communicating current version
2021-04-26 21:04:16,165 INFO    MainThread:2878 [wandb_init.py:init():489] got version response 
2021-04-26 21:04:16,166 INFO    MainThread:2878 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-26 21:04:16,892 INFO    MainThread:2878 [wandb_init.py:init():522] starting run threads in backend
2021-04-26 21:04:17,803 INFO    MainThread:2878 [wandb_run.py:_console_start():1524] atexit reg
2021-04-26 21:04:17,803 INFO    MainThread:2878 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-26 21:04:17,803 INFO    MainThread:2878 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-26 21:04:17,804 INFO    MainThread:2878 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-26 21:04:17,804 INFO    MainThread:2878 [wandb_init.py:init():546] run started, returning control to user process
2021-04-26 21:04:44,682 INFO    MainThread:2878 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 255
2021-04-26 21:04:44,682 INFO    MainThread:2878 [wandb_run.py:_restore():1466] restore
2021-04-26 21:04:47,407 INFO    MainThread:2878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 826
  total_bytes: 826
}

2021-04-26 21:04:47,907 INFO    MainThread:2878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 826
  total_bytes: 3010
}

2021-04-26 21:04:48,488 INFO    MainThread:2878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 826
  total_bytes: 6434
}

2021-04-26 21:04:48,593 INFO    MainThread:2878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 826
  total_bytes: 6434
}

2021-04-26 21:04:48,697 INFO    MainThread:2878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 826
  total_bytes: 6434
}

2021-04-26 21:04:48,799 INFO    MainThread:2878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 826
  total_bytes: 6434
}

2021-04-26 21:04:48,903 INFO    MainThread:2878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6434
  total_bytes: 6434
}

2021-04-26 21:04:49,006 INFO    MainThread:2878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6434
  total_bytes: 6434
}

2021-04-26 21:04:49,110 INFO    MainThread:2878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6434
  total_bytes: 6434
}

2021-04-26 21:04:49,228 INFO    MainThread:2878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6434
  total_bytes: 6434
}

2021-04-26 21:04:49,332 INFO    MainThread:2878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6434
  total_bytes: 6434
}

2021-04-26 21:04:49,436 INFO    MainThread:2878 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6434
  total_bytes: 6434
}

2021-04-26 21:04:50,530 INFO    MainThread:2878 [wandb_run.py:_show_files():1838] logging synced files
